Lombiq.Tests.UI/Services/SynchronizingWebApplicationSnapshotManager.cs

```diff
@@ -46,7 +46,6 @@ public async Task<Uri> RunOperationAndSnapshotIfNewAsync(AppInitializer appIniti
 
             var result = await appInitializer();
             await result.Context.Application.TakeSnapshotAsync(_snapshotDirectoryPath);
-            await result.Context.Application.ResumeAsync();
```

Comment:
Note to myself: This was a question mark, but the instance is resumed in `TakeSnapshotInnerAsync`.

Reply:
Yeah, previously `TakeSnapshotInnerAsync` lacked the resume, but since it stops the instance, it's proper to have the resume there and not force the consumer to do it (after discovering that the instance was unexpectedly stopped).